README.md

```diff
@@ -1 +1 @@
-./packages/disploy/README.md
\ No newline at end of file
+./packages/disploy/README.md
```

Comment:
Please revert the changes on the symlink, formatting it; breaks it. Needs to be added to the prettier ignore later down the road.